public/robots.txt

```diff
@@ -0,0 +1,3 @@
+# Allow all crawlers
+User-agent: *
+Allow: /
```

Comment:
If we have a sitemap.xml (not sure if we do), we can add it in here 
as well (or in the future), e.g. https://opensauced.pizza/sitemap.xml

```suggestion
Sitemap: https://opensauced.pizza/sitemap.xml

# Allow all crawlers
User-agent: *
Allow: /
```